### PR TITLE
[WIP] Update to use ./kompose instead of $GOPATH kompose binary

### PR DESCRIPTION
**Pull request by cdrage**
The issue is that when running `make test-cmd` you'll have to cp kompose
to the /usr/local/bin directory or the $GOPATH directory.

Since `make test` is being used consistently in the same git diretory,
it makes sense to use the local-built binary.

### CODE REVIEW

**Comment by cdrage** [comment]
Blocked on #770 since `kompose version` does not fail if no kompose is located.

File: .travis.yml [hunk 1]
```diff
@@ -4,9 +4,6 @@ sudo: required
 
 language: go
 
-# make this also work for forks
-go_import_path: github.com/kubernetes/kompose
```
Comment (kadel) [inline .travis.yml]: Why is this removed? It shouldn't have any effect on this, or does it?

Comment (cdrage) [reply .travis.yml]: Yeah, I need to re-work this...

**Comment by cdrage** [comment]
Closing for now.
